samples/src/test/java/org/autorefactor/refactoring/rules/samples_out/AndroidRecycleSample.java

```diff
@@ -0,0 +1,141 @@
+package org.autorefactor.refactoring.rules.samples_out;
+
+import android.content.ContentProvider;
+import android.content.ContentProviderClient;
+import android.content.ContentResolver;
+import android.content.res.TypedArray;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.net.Uri;
+import android.os.Message;
+import android.os.RemoteException;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.view.VelocityTracker;
+import android.view.View;
+import android.content.Context;
+import android.os.Parcel;
+
+public class AndroidRecycleSample {
+    public void cursorError1(SQLiteDatabase db, long route_id) {
+        Cursor cursor = db.query("TABLE_TRIPS",
+                new String[]{ "KEY_TRIP_ID" },
+                "ROUTE_ID=?",
+                new String[]{Long.toString(route_id)},
+                null, null, null);
+        cursor.close();
```

Comment:
I don't know the reason but it is not common in Android db queries:
https://www.codota.com/android/methods/android.database.sqlite.SQLiteDatabase/query